README.md

```diff
@@ -1067,6 +1067,7 @@ Games, game servers and control panels.
 - [edX](https://www.edx.org/) - The Open edX platform is open-source code that powers edX.org. ([Source Code](https://github.com/edx/)) `AGPL-3.0` `Python`
 - [Gibbon](https://www.gibbonedu.org/) - The flexible, open source school management platform designed to make life better for teachers, students, parents and leaders. ([Source Code](https://github.com/GibbonEdu/core)) `GPL-3.0` `PHP`
 - [ILIAS](https://www.ilias.de) - ILIAS is the Learning Management System that can cope with anything you throw at it. ([Demo](https://demo.ilias.de), [Source Code](https://github.com/ILIAS-eLearning/ILIAS)) `GPL-3.0` `PHP`
+- [LernTools](https://lerntools.org/) - Tools for digital teaching with a focus on privacy. Components: ABCD-Quiz (comparable with Kahoot!), ideas (comparable with Padlet), survey ([Source Code](https://codeberg.org/lerntools/))  `GPL-3.0` `Javascript, Nodejs`
```

Comment:
- Please check the relevant boxes in the pull request description
- Please replace `, ` with `/` in the language tag
- No landing page in English?
- Missing `.` at the end of the description
- Where is the installation documentation?